packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -105,7 +105,6 @@ contract L1Block is ISemver {
                 mstore(0x00, 0x3cc50b45) // 0x3cc50b45 is the 4-byte selector of "NotDepositor()"
                 revert(0x1C, 0x04) // returns the stored 4-byte selector from above
             }
-            let data := calldataload(4)
```

Comment:
Weird, any idea why this was here?

Reply:
No idea

Reply:
Remnant from a previous implementation where we were not reusing the slot.

Reply:
Solidity linter isn't able to catch this?

Reply:
probably no compiler warnings in assembly.